notebooks/03-query/11-joins.ipynb

```diff
@@ -1479,6 +1479,82 @@
     "print_timestamp()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "fcba25f2-3116-4b41-965a-8acdb00c8651",
+   "metadata": {},
+   "source": [
+    "## New in Druid 29.0.0 - INNER JOIN with arbitrary conditions\n",
+    "\n",
+    "Before the 29.0.0 release, all join conditions had to be equalities. With the new functionality inequalities can also be used. \n",
```

Comment:
I thought that we ended on, let's signal the new features within the notebooks rather than new notebooks for new features. My thought is that this makes it very clear when they look at the notebook outline, users can immediately see what is new.

Reply:
Done